File: Programming Basics/C++/Conditional Statements/SumSeconds/SumSeconds.cpp
#include <iostream>
using namespace std;

int main()
{
    int firstTime, secondTime, thirdTime;
    cin >> firstTime >> secondTime >> thirdTime;

    int total = firstTime + secondTime + thirdTime;

    int minutes = total / 60, seconds = total % 60;

    // First approach:
    // if (seconds < 10) cout << minutes << ":0" << seconds << endl;
    // else cout << minutes << ":" << seconds << endl;

    // Second approach:
    // cout << minutes << ":";
    // if (seconds < 10) cout << "0";
    // cout << seconds << endl;
    
    // Third approach:
    cout << minutes << ":" << string(seconds < 10, '0') << seconds << endl;
}

File: Programming Basics/JavaScript/Conditional Statements/Exercises/sum-seconds.js
function solve(input) {
    const firstTime = Number(input[0]);
    const secondTime = Number(input[1]);
    const thirdTime = Number(input[2]);

    const total = firstTime + secondTime + thirdTime;

    const seconds = total % 60;
    const minutes = (total - seconds) / 60;

    if (seconds < 10) console.log(`${minutes}:0${seconds}`);
    else console.log(`${minutes}:${seconds}`);
}
